www/collision.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<head>
+  <style>
+    #balls {
+      position:absolute;
+      top: 0px;
+      left:0px;
+    }
+  </style>
+</head>
+<body>
+  <script src="js/d3.v3.min.js"></script>
+  <script src="./../build/webgazer.js"></script>
```

Comment:
I feel like we should double check this with Alex in tomorrow's meeting if that path is okay -- but I wouldn't know how else to do it otherwise!